cmd/agent/main.go

```diff
@@ -0,0 +1,185 @@
+package main
```

Comment:
Do we need license headers here?

Reply:
Yes. Want to get your first commit to this repo?